independent-projects/arc/runtime/src/main/java/io/quarkus/arc/impl/Reflections.java

```diff
@@ -176,6 +178,18 @@ public static Object invokeMethod(Class<?> clazz, String name, Class<?>[] paramT
         }
     }
 
+    public static <T> T getEnumConstant(String name, Class<T> clazz) {
```

Comment:
I don't think it makes sense to add this in ArC. Better keep it in the substitution class.